tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html

```diff
@@ -179,6 +179,9 @@
         smoothingWeight: Number,
         tagMetadata: Object,
         tooltipSortingMethod: String,
+        // Function that takes 2 string arguments (tag and run) and returns a
+        // string URL for fetching data.
+        dataUrl: Function,
```

Comment:
Maybe let's call it dataUrlFn just to be a little clearer that it's a Function, versus e.g. a string or literal URL?

Reply:
Makes sense. I was trying to be consistent: the same property in tf-line-char-data-loader is called dataUrl.

Reply:
Re: "yeah I guess the hparams plugin serves the metric data over its own endpoint rather than just relying on the scalars plugin, so this parameter is necessary for that reason?"

Yes, precisely. And when we integrate with Vizier, there won't be any scalar's end-point.

Reply:
Ah, you're right about line-chart-data-loader.  Perhaps it's better as you had it originally then.